lite/tests/classification/test_accuracy.py

```diff
@@ -84,10 +84,10 @@ def test_accuracy_basic(basic_classifications: list[Classification]):
         "missing_prediction_labels": [],
     }
 
-    metrics = evaluator.evaluate(score_thresholds=[0.25, 0.75])
+    metrics = evaluator.evaluate(score_thresholds=[0.25, 0.75], as_dict=True)
 
     # test Accuracy
-    actual_metrics = [m.to_dict() for m in metrics[MetricType.Accuracy]]
+    actual_metrics = [m for m in metrics[MetricType.Accuracy]]
```

Comment:
nit: it's probably cleaner to pass Accuracy to `metrics_to_return`, then you don't need this step at all

Reply:
The alt would be 

```
metrics = evaluate.evaluate(
    ...,
    metrics_to_return=[MetricType.Accuracy],
)[MetricType.Accuracy]

actual_metrics = [m for m in metrics]
```

which im not a huge fan of visually